jquery.flot.touchNavigate.js

```diff
@@ -194,8 +194,17 @@
 
     function getAxis(plot, e, gesture, navigationState) {
         if (e.detail.type === 'touchstart') {
```

Comment:
Could you please rewrite these if-if-else statements without looking to the original event.type?
Ideally the touchNavigate plugin should be decoupled from the touch plugin. In theory anyone else should be able to emit high level events like 'pinch' or 'pan' without faking the native 'touchstart', 'touchmove' or 'touchend' events.